test/Microsoft.ML.CodeGenerator.Tests/ApprovalTests/ConsoleCodeGeneratorTests.AzureCodeGeneratorTest.test.ConsoleApp.csproj.approved.txt

```diff
@@ -7,6 +7,8 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.ML" Version="StablePackageVersion" />
     <PackageReference Include="Microsoft.ML.OnnxTransformer" Version="StablePackageVersion" />
+    <PackageReference Include="Microsoft.ML.OnnxRuntime" Version="1.2.3" />
+    <PackageReference Include="Microsoft.ML.CodeGenerator.Helper" Version="1.2.3" />
```

Comment:
Why a pinned version for `Microsoft.ML.OnnxRuntime` & `Microsoft.ML.CodeGenerator.Helper`?

Reply:
Because Microsoft.ML.OnnxTransformer 1.5.0 requires users to indicate the specific onnxruntime package (CPU or GPU)

Reply:
Not any more
### Resolved